index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <div class="profile-pic">
+                <img src="assets/Anamika.R.S.jpg" alt="profile-pic" class="pic">
+            </div>
+        
+            <div class="name">
+                <h1 class="my-name">Anamika R S</h1>
+                <p id="position" class="my-position">Junior Software Engineer | Product Development</p>
+            </div>
+        </header>
+        <nav>
+
+            
+            <label for="menu-toggle" class="burger">
+                <span></span>
+                <span></span>
+                <span></span>
+            </label>
+            <input  class="checkbox" type="checkbox" id="menu-toggle">
+            <div class="navigation">
+                <nav class="navbar">
+                    <ul>
+                        
+                        <li>
+                            <a class="nav-link" href="#aboutme">About Me</a>
+                        </li>
+                        <li>
+                            <a class="nav-link" href="#myexp">Work experience</a>
+                        </li>
+                        <li>
+                            <a class="nav-link" href="#myeducation">Education</a>
+                        </li>
+                        <li>
+                            <a class="nav-link" href="#mycontact">Contact</a>
+                        </li>
+                        <li>
+                            <a class="nav-link" href="#myresume">Resume</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+
+        </nav>
+
+        
+        <article class="main">
+
+            <div class="About" id="aboutme">
+                <h1>About Me</h1>
+                <hr><br>
+                <p>I am a B.Tech Graduate In Information Technology. </p><br>
+                <p>Currently working as a Junior Software Engineer at SEEMYMACHINES.</p><br>
+                <p>Highly passionate about Web designing.</p><br>
+            </div>
+        
+            <div class="Experience" id="myexp">
+                <h1>Work Experience</h1><hr><br>
+                <p>Junior Software Engineer</p><br>
+                <p>SeeMyMachines .Full-Time</p><br>
+                <p>Jun 2024 - present</p><br>
+            </div>
+        
+            <div class="Education" id="myeducation">
```

Comment:
Use a uniform casing - Camel casing, snake case, etc for naming classes and ids.